packages/@aws-cdk/core/README.md

```diff
@@ -246,24 +246,32 @@ new CustomResource(this, 'MyMagicalResource', {
 ### Custom Resource Providers
 
 Custom resources are backed by a **custom resource provider** which can be
-implemented in one of the following ways (ordered from low-level to high-level):
+implemented in one of the following ways. The following table compares the
+various provider types (ordered from low-level to high-level):
 
-* `@aws-cdk/aws-sns.Topic`
-* `@aws-cdk/aws-lambda.Function`
-* `@aws-cdk/custom-resources.Provider`
+| Provider                                                             | Compute Type | Error Handling | Submit to CloudFormation | Max Timeout     | Language | Footprint |
```

Comment:
Nice! I like this table.

It's not clear what 'footprint' here means. Clarify above the table or use a different phrasing.

Reply:
Added.